packages/orbit-components/playwright/index.tsx

```diff
@@ -2,10 +2,25 @@ import "./index.css";
 
 import * as React from "react";
 import { beforeMount } from "@playwright/experimental-ct-react/hooks";
+import { tokensToCssVars } from "@kiwicom/orbit-design-tokens";
 
 import RandomIdProvider from "../src/OrbitProvider/RandomId/Provider";
+import { getCssVarsForWL } from "../src/OrbitProvider";
+import defaultTheme from "../src/defaultTheme";
 
 beforeMount(async ({ App }) => {
+  const ID = "orbit-theme-css-vars";
+  const style = document.createElement("style");
+  style.setAttribute("id", ID);
+  style.innerText = tokensToCssVars({
+    tokens: getCssVarsForWL(defaultTheme.orbit),
+    cssClass: ":root",
+  });
+  const head = document.getElementsByTagName("head")[0];
+  if (!document.getElementById(ID)) {
```

Comment:
Do you know if we need this check? Not 100% sure from the docs, but it looks like the `beforeMount` hook runs only once.

https://playwright.dev/docs/test-components#hooks

Reply:
Yes, it's not needed. Well spotted